include/wx/msw/radiobut.h

```diff
@@ -13,7 +13,7 @@
 
 #include "wx/msw/ownerdrawnbutton.h"
 
-class WXDLLIMPEXP_CORE wxRadioButton : public wxMSWOwnerDrawnButton<wxControl>
+class WXDLLIMPEXP_CORE wxRadioButton : public public wxRadioButtonBase<wxMSWOwnerDrawnButton<wxControl>>
```

Comment:
```suggestion
class WXDLLIMPEXP_CORE wxRadioButton : public wxRadioButtonBase< wxMSWOwnerDrawnButton<wxControl> >
```
(there are 2 changes here: fix the extra `public` and add a space for C++98 compatibility)

Reply:
> Thanks, no real comments, but we need to fix the build, of course, and we also need to document the new functions in `interface/wx/radiobut.h`. Ideal would be to also add some unit tests for them, but I guess we can live without them as this is not really new code...

Thanks, yes, I overlooked the Win builds ...